app/views/hyrax/file_sets/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container-fluid">
```

Comment:
Likewise to the other comment, does this belong in the Fileset view, or the [base show partial](https://github.com/samvera/hyrax/blob/master/app/views/hyrax/base/show.html.erb) for the object itself, with the representative media partial perhaps moved from it's default location to a more prominent place? (that's also something we could sort out later w/ usability study)

Reply:
Yes, it sounds like something that we can could sort out later. I just override it for the OpenSeadragon viewer to be display properly for now.

Reply:
Yeah, agreed. The exact placement in the view we can sort out later 👍 I just want to make sure we're housing it in the correct partial.